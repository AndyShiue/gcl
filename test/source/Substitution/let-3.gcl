var y : Int 
let P x = x = 3

y := y + 1

{ P y }
