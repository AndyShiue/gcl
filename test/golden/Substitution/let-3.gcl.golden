( 0: P (y + 1)
  P (y + 1) ===> y + 1 = 3
    
, (0, [[reduce], [subst], [reduce-App-Expand-Lam], [reduce]] , P (y + 1)) )